library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java

```diff
@@ -464,17 +517,35 @@ public void setWeekDayTextAppearance(int resourceId) {
     }
 
     /**
-     * @return the currently selected day, or null if no selection
+     * @return the selected day, or null if no selection. If in multiple selection mode, this
+     * will return the last selected date
      */
     public CalendarDay getSelectedDate() {
```

Comment:
@ekchang I would agree the name would be better, but I think it should stay the same for backwards compatibility. As it stands, the only API that's changing is the selection callback; I'd like to keep it that way. And if you're using this, it's probably in single selection mode, since you'd want many dates in multiple mode.